sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/XmlExternalEntityShouldNotBeParsed.cs

```diff
@@ -216,7 +216,7 @@ private static bool IsAllowedValueForXmlTextReader(object constantValue)
                     symbol.Kind == SymbolKind.Method &&
                     symbol.ContainingType.GetSymbolType().IsAny(UnsafeXmlResolvers);
 
-            private static bool IsAllowedObject(ISymbol symbol) =>
+            private static bool IsAllowedObject(ISymbol symbol, SyntaxNode node, SemanticModel model) =>
```

Comment:
I'd comment that we don't use `node` and `model` on purpose (to avoid confusion in the future)